src/cider/nrepl/middleware/stacktrace.clj

```diff
@@ -110,7 +110,7 @@
 
 (defn flag-tooling
   "Walk the call stack from top to bottom, flagging frames below the first call
-  to `clojure.lang.Compiler` or `clojure.tools.nrepl.*` as `:tooling` to
+  to `clojure.lang.Compiler` or `nrepl.*` as `:tooling` to
   distinguish compilation and nREPL middleware frames from user code."
   [frames]
   (let [tool-regex #"^clojure\.lang\.Compiler|^clojure\.tools\.nrepl|^cider\."
```

Comment:
Shouldn't the `clojure\.tools\.nrepl`  in the regex also be changed to just `nrepl`?

Reply:
Yeah, it should be changed.